components/brave_ads/core/internal/ads_impl.cc

```diff
@@ -253,44 +253,40 @@ void AdsImpl::GetDiagnostics(GetDiagnosticsCallback callback) {
   DiagnosticManager::GetInstance().GetDiagnostics(std::move(callback));
 }
 
-mojom::UserReactionType AdsImpl::ToggleLikeAd(const base::Value::Dict& value) {
-  return is_initialized_ ? AdHistoryManager::GetInstance().LikeAd(
```

Comment:
Maybe we should run the callback with `mojom::UserReactionType::kNeutral` if `is_initialized_` is false? The same for other reactions callbacks.

Reply:
Fixed, thanks